assets/test/view_test.js

```diff
@@ -225,6 +225,13 @@ describe("View + DOM", function(){
     view = new View(liveViewDOM(html), liveSocket)
     view.joinCount = 2
     expect(view.formsForRecovery().length).toBe(0)
+
+    html = "<form id='my-form' phx-change='[[\"push\",{\"event\":\"update\",\"target\":1}]]'><input name=\"foo\" /></form>"
+    view = new View(liveViewDOM(html), liveSocket)
+    view.joinCount = 1
+    const newForms = view.formsForRecovery(html)
+    expect(newForms.length).toBe(1)
+    expect(newForms[0][0].getAttribute('phx-change')).toBe('[[\"push\",{\"event\":\"update\",\"target\":1}]]')
```

Comment:
The test for the CSS.escape approach would need something like

Something like

```js
global.CSS = {
  escape: jest.spy().mockImplementation(v => v.replaceAll(/([\[\]"])/g, '\\$1'))
}
```
 before line 229

then 

```js
delete global CSS 
```

after line 234